### PR TITLE
chore: trigger readthedocs preview only when docs change

### PR DESCRIPTION
**Pull request by huxuan**
Test:
- https://github.com/huxuan/ss-python/pull/30
- https://github.com/huxuan/ss-python/pull/31
- https://github.com/huxuan/ss-python/pull/32

Actually, I am a little confused about the preview link in this pull request. Seems the changes to workflows does not take effect immediately. Let me know if I made anything wrong.

----
📚 Documentation preview 📚: https://ss-python--377.org.readthedocs.build/en/377/

### CODE REVIEW

**Review by msclock — APPROVED** [review]

File: .github/workflows/readthedocs-preview.yml [hunk 1]
```diff
@@ -13,5 +13,7 @@ on:
   pull_request_target:
     types:
       - opened
+    paths:
+      - 'docs/**'
```
[the remark this answers is not in the document]
Comment (huxuan) [reply .github/workflows/readthedocs-preview.yml]: May I know the configuration you specified? Actually, I failed to find a configuration for similar purpose here: https://docs.readthedocs.io/en/stable/config-file/v2.html

[the remark this answers is not in the document]
Comment (msclock) [reply .github/workflows/readthedocs-preview.yml]: my understanding is the sphinx configuration file charges in the root of repo should also be detected by readthedocs/preview action.

[the remark this answers is not in the document]
Comment (huxuan) [reply .github/workflows/readthedocs-preview.yml]: Sorry for my misunderstood. Thanks for pointing out. Will fix soon.

[the remark this answers is not in the document]
Comment (huxuan) [reply .github/workflows/readthedocs-preview.yml]: Besides the `.readthedocs.yaml` file, I also add the `.github/workflows/readthedocs-preview.yml` for similar reason. PTAL. :-)

**Review by msclock — APPROVED** [review]

**Pull request merged**